src/components/movie-items/mockMovies.tsx

```diff
@@ -1,47 +1,23 @@
 import { MovieCard } from 'components/movie-card/movie-card';
-import movie1 from '../../assets/Movie1.jpg';
+import { useMovieContext } from 'contexts/movie-context';
 
 import styles from './movie-items.module.scss';
 
 export const MockMoves = () => {
+  const { movies, setSelectedMovie } = useMovieContext();
+
   return (
     <div className={styles.movies}>
-      <MovieCard
-        img={movie1}
-        title="Pulp Fiction"
-        year={2004}
-        description="Action & Adventure"
-      />
-      <MovieCard
-        img={movie1}
-        title="Pulp Fiction"
-        year={2004}
-        description="Action & Adventure"
-      />
-      <MovieCard
-        img={movie1}
-        title="Pulp Fiction"
-        year={2004}
-        description="Action & Adventure"
-      />
-      <MovieCard
-        img={movie1}
-        title="Pulp Fiction"
-        year={2004}
-        description="Action & Adventure"
-      />
-      <MovieCard
-        img={movie1}
-        title="Pulp Fiction"
-        year={2004}
-        description="Action & Adventure"
-      />
-      <MovieCard
-        img={movie1}
-        title="Pulp Fiction"
-        year={2004}
-        description="Action & Adventure"
-      />
+      {movies.map((movie) => (
+        // eslint-disable-next-line react/jsx-key
```

Comment:
Strongly advise you to add `key` attribute to avoid issue with React list processing under the hood to prevent unexpected behaviour. You may create your `key` by several props of `movie` to make it unique